sklearn/isotonic.py

```diff
@@ -226,11 +235,20 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     array([1.8628..., 3.7256...])
     """
 
-    def __init__(self, *, y_min=None, y_max=None, increasing=True, out_of_bounds="nan"):
+    def __init__(
+        self,
+        *,
+        y_min=None,
+        y_max=None,
+        increasing=True,
+        out_of_bounds="nan",
+        centered=False,
```

Comment:
How should we name it?
- `centered` to be in line with the paper?
- `strict` or `strict_interpolation` to better say what it actually does?

Reply:
I personally like `strict`, because it is more descriptive. However, as you point out, it is not in line with the paper, and it is also implemented in an R package as `centered`[1], so more people would recognize that. I will happily let you decide on the name.

[1] https://cran.r-project.org/web/packages/cir/cir.pdf

Reply:
I also prefer the more telling term `strict`.

Reply:
I have renamed the parameter to `strict`.